compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/diagnostics/LightTreePositioningStrategies.kt

```diff
@@ -317,6 +341,27 @@ private fun FlyweightCapableTreeStructure<LighterASTNode>.receiverTypeReference(
     }
 }
 
+private fun FlyweightCapableTreeStructure<LighterASTNode>.defaultValue(node: LighterASTNode): LighterASTNode? {
```

Comment:
Please don't say we can simply use `lastChild` for retrieving the default value. :)

Here I attempt to make sure necessary pieces--- colon, followed by type reference, followed by = ---are indeed there in _order_.

Reply:
I'd say it may be easier to travel children from the opposite side...

Reply:
Alright, `reversed` and make sure we won't pick any other parts, e.g., white space, colon, eq, type reference, and identifier. It will pick the last child immediately if default value is there.